Stanford/JumpstartEngineering/Install/Layouts/PicalPeople.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @file
+ * Configure PICAL stanford_person node display.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Layouts;
+/**
+ *
+ */
+class PicalPeople extends \AbstractInstallTask {
```

Comment:
Since we're just doing a `variable_set()`, I'm leaning towards not setting requirements.